cypress/e2e/pages/mediaAssetPage/testsForAMPOnly.js

```diff
@@ -1,10 +1,7 @@
 /* eslint-disable import/prefer-default-export */
 
 // For testing features that may differ across services but share a common logic e.g. translated strings.
-export const testsThatFollowSmokeTestConfigForAMPOnly = ({
-  service,
-  pageType,
-}) => {
+export default ({ service, pageType }) => {
```

Comment:
On line 8: Any test which refers to `Cypress.env('currentPath')}` will no longer work - we need to pass in `path` as a parameter and use it instead